rails.md

```diff
@@ -6,28 +6,44 @@ Answer the following questions. First, without external resources. Challenge you
 1. MVC (Model View Controller) is a pattern for the architecture of a software program. Give a brief description of each component and describe how Ruby on Rails handles MVC.
 
   Your answer:
+  
+  Model (Active Record) handles relationships between objects and databases.
+View presents the data in the particular format determined by the controller.
+Controller organizes data to fit the needs of the view, and directs the traffic within the controller.
+
 
   Researched answer:
+  
+The Model View Controller principle divides the work of an application into three separate but closely cooperative subsystems.
+Model – Active Record 
+
+Model:
+This subsystem is implemented in ActiveRecord library, which provides an interface and binding between the tables in a relational database and the Ruby program code that manipulates database records. Ruby method names are automatically generated from the field names of database tables.
+View:
+This subsystem is implemented in ActionView library, which is an Embedded Ruby (ERb) based system for defining presentation templates for data presentation. Every Web connection to a Rails application results in the displaying of a view.
+Controller:
+
+This subsystem is implemented in ActionController, which is a data broker sitting between ActiveRecord (the database interface) and ActionView (the presentation engine).
 
 
 
 2. Using the information given, fill in the blanks to complete the steps for creating a new view in a Rails application.
 
-  Step 1: Create the _________ in the file config/routes
+  Step 1: Create the ______route___ in the file config/routes
   ```
   get '/about' => 'statics#about'
   ```
 
-  Step 2: Create the ____________ in the file _____________
+  Step 2: Create the _____method_______ in the file _____controllers________
   ```
-  class ____________ < ApplicationController
-    def ___________
-      render: ________________
+  class ___statics_________ < ApplicationController
```

Comment:
Step 2: Create the controller in the file app/statics_controller.rb

class StaticsController < ApplicationController